### DISCUSSION TITLE
Quick tags on Drafts

### DISCUSSION THREAD

**Discussion by barbex** [AprilSylph/XKit-Rewritten#653] [Help Desk] [answered]
PC, Firefox.
Trying to use the Quick Tags symbol in tumblr drafts lets me put in a list of words but when I hit enter, I get the message "Uh-oh! That didn't work. Please try again." at the bottom of the screen. This has been the case for several weeks. I hope it can be fixed because it's one of the most useful hacks for me.

**Comment by AprilSylph** [comment, marked as answer]
This happens if the draft was made using the legacy editor. Quick Tags only works on NPF posts (those made using mobile or the Beta editor on desktop); see #411.
